cpp/core/CMakeLists.txt

```diff
@@ -186,8 +186,11 @@ set(SPARK_COLUMNAR_PLUGIN_SRCS
         operators/c2r/ColumnarToRow.cc
         shuffle/reader.cc
         shuffle/ArrowShuffleWriter.cc
-        shuffle/CelebornShuffleWriter.cc
         operators/writer/ArrowWriter.cc
+        shuffle/PartitionWriter.cc
+        shuffle/rss/RemotePartitionWriter.cc
+        shuffle/LocalPartitionWriter.cc
+        shuffle/rss/CelebornPartitionWriter.cc
```

Comment:
Please pay attention to the file order, move rss to sequece order

Reply:
@jinchengchenghh done